### ISSUE TITLE
Historical dataset error

### ISSUE THREAD

**Issue by robmarkcole** [IsaacCorley/pytorch-enhance#2]
On running the pytorch lightning example but swapping in the Historical dataset, the following error is raised:

```
RuntimeError: Given groups=1, weight of size [64, 3, 9, 9], expected input[1, 1, 256, 256] to have 3 channels, but got 1 channels instead
```

I assume there is a trivial fix in the dataset?

**Comment by isaaccorley** [comment]
The Historical dataset is grayscale so it would require changing the number of in_channels on the first conv layer of the model you are using. Looks like I hardcoded it but I can make a fix to define the number of input channels.

Edit: and change the number of out_channels on the last conv layer.

**Comment by isaaccorley** [comment]
Fixed amongst other updates in #3.

You can define input/output channels of a model like

```python
from torch_enhance.models import SRResNet
model = SRResNet(scale_factor=2, channels=3) # channels=1 for grayscale
```

Although if you train a model on a RGB dataset you won't be able to transfer to grayscale dataset unfortunately unless you do tile the grayscale channel to 3 channels (which will likely give mixed results).

**Comment by robmarkcole** [comment]
Perfect, thanks for the quick fix!
